GiniVision/Classes/OpenWithTutorialCollectionCell.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  OpenWithTutorialCollectionCell.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/23/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+final class OpenWithTutorialCollectionCell: UICollectionViewCell {
+    
+    let padding:(top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat) = (40, 20, 40, 20)
```

Comment:
It refers to the cell itself, since if you call the cell in some place and you want to know the paddign it would be `cell.padding` and not `cell.cellPadding`.

Reply:
I'm not sure the cell should set its own padding. But ok...